packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -188,30 +212,10 @@ const BuySellModal = ({ table_type, selected_ad, should_show_popup, setShouldSho
     const setSubmitForm = submitFormFn => (submitForm.current = submitFormFn);
 
     const has_rate_changed =
-        (!!error_message && buy_sell_store.form_error_code !== api_error_codes.ORDER_CREATE_FAIL_RATE_CHANGED) ||
+        (!!error_message && buy_sell_store.form_error_code === api_error_codes.ORDER_CREATE_FAIL_RATE_CHANGED) ||
         has_rate_changed_recently;
     const onSubmit = has_rate_changed ? onSubmitWhenRateChanged : submitForm.current;
 
-    React.useEffect(() => {
```

Comment:
Moved the same code to line 123 since its a duplicate `useEffect(() => {...}, [])`